src/transformers/utils/hub.py

```diff
@@ -762,6 +764,7 @@ def push_to_hub(
         safe_serialization: bool = True,
         revision: str = None,
         commit_description: str = None,
+        tags: List[str] = None,
```

Comment:
```suggestion
        tags: Optional[List[str]] = None,
```

`Optional` since default value is None?